parser/v2/parser.go

```diff
@@ -93,17 +96,11 @@ func (p *Parser) ParseTransactions(traces []byte, tipset *types.ExtendedTipSet,
 		// Only process sub-calls if the parent call was successfully executed
 		if trace.ExecutionTrace.MsgRct.ExitCode.IsSuccess() {
 			subTxs := p.parseSubTxs(trace.ExecutionTrace.Subcalls, trace.MsgCid, tipset, ethLogs,
-				trace.Msg.Cid().String(), transaction.Id, 0)
+				trace.Msg.Cid().String(), transaction.Id, 0, trace.GasCost)
 			if len(subTxs) > 0 {
 				transactions = append(transactions, subTxs...)
 			}
 		}
-
-		// Fees
-		if trace.GasCost.TotalCost.Uint64() > 0 {
-			feeTx := p.feesTransactions(trace, tipset, transaction.TxType, transaction.Id)
-			transactions = append(transactions, feeTx)
-		}
```

Comment:
Fees were created only at level 0 tx. I think new changes attempt to add those values on subcalls too. I guess we should not. Only level 0 tx should have fees data field with some real data